app.js

```diff
@@ -0,0 +1,98 @@
+var app = angular.module('PathOfDamage', []);
+app.controller('Damage', function ($scope) {
+  $scope.hits = [100, 500, 1000, 2000, 3000, 4000, 5000, 7500, 10000];
+
+  $scope.buffs = {
+    total: 0,
+    values: []
+  };
+  $scope.shifts = {
+    total: 0,
+    values: []
+  };
+  $scope.damageReductions = {
+    total: 0,
+    values: []
+  };
+  $scope.flatTaken = {
+    total: 0,
+    values: []
+  };
+  $scope.reducedTaken = {
+    total: 0,
+    values: []
+  };
+  $scope.lessTaken = {
+    total: 0,
+    values: []
+  };
+
+  $scope.armor = "";
```

Comment:
You can probably declare this as null, since you think of it as a number rather than a string.